xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/java/org/xwiki/officeimporter/internal/server/DefaultOfficeServer.java

```diff
@@ -137,6 +138,7 @@ public void initialize() throws OfficeServerException
             ExternalOfficeManager.Builder externalProcessOfficeManager = ExternalOfficeManager.builder();
             externalProcessOfficeManager.portNumbers(this.config.getServerPorts());
             externalProcessOfficeManager.connectOnStart(true);
+            externalProcessOfficeManager.hostName(config.getServerHost());
```

Comment:
Are we sure that `12.7.0.0.1` is the default? (vs `localhost` for example).

Another option is to set the a host only if not empty in the XWiki config.

Reply:
I confirm that `127.0.0.1` is the default -> https://github.com/jodconverter/jodconverter/blob/master/jodconverter-local/src/main/java/org/jodconverter/local/office/ExternalOfficeManager.java#L60